Indicators/KeltnerChannels.cs

```diff
@@ -118,7 +118,7 @@ protected override decimal ComputeNextValue(IBaseDataBar input)
             AverageTrueRange.Update(input);
 
             var typicalPrice = (input.High + input.Low + input.Close)/3m;
-            MiddleBand.Update(input.Time, typicalPrice);
+            MiddleBand.Update(input.EndTime, typicalPrice);
```

Comment:
Let's please add some unit test around the change, would be nice to assert the indicator compared with another source like trading view